apps/backend/.eslintrc.js

```diff
@@ -16,5 +16,6 @@ module.exports = {
     '@typescript-eslint/no-empty-function': 'off',
     '@typescript-eslint/no-extra-semi': 'off',
     '@typescript-eslint/no-unused-vars': 'error',
+    'ordered-imports': [true, { 'grouped-imports': false }], //disable anoying empty line between import groups
```

Comment:
![image](https://github.com/kir-dev/BeAlcoholic/assets/13004605/4d0be18e-2fca-4894-b419-aba0bfdeb661)
I guess we should use 2 instead of true and maybe 0 instead of false

Reply:
rather deleted it because it caused more problems and is not as annoying as I thought before